Crea readme.md

fiorellam opened the pull request.

fiorellam committed:
Crea readme.md

devarops reviewed (changes requested):
Por favor despelusa el README usando un _linter_ para Markdown. Yo uso [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).

fiorellam committed:
Edita README con ayuda de markdownlint

devarops reviewed (changes requested):
A veces hablas "de usted": _seleccione_, _ejecute_, _abra_; pero al final hablas "de tú": _escribe_.

Por favor usa una sóla forma. Yo prefiero  _selecciona_, _ejecuta_, _abre_ ya que así nos hablamos entre nosotros, sin embargo, tú puedes elegir la forma que prefieras, pero sólo una.

devarops commented:
A veces hablas "de usted": _seleccione_, _ejecute_, _abra_; pero al final hablas "de tú": _escribe_.

Por favor usa una sóla forma. Yo prefiero  _selecciona_, _ejecuta_, _abre_ ya que así nos hablamos entre nosotros, sin embargo, tú puedes elegir la forma que prefieras, pero sólo una.

devarops commented on README.md:
```diff
@@ -0,0 +1,46 @@
+* En la ventana de **new project** seleccione Java.
+* Seleccione una opción en **project sdk**, en caso de que no aparezca ninguna opción, da click en **new** y especifique la ruta en la que se encuentra el directorio del jdk de Java.
+Ejemplo: C:\Program Files\Java\jdk
+
```
Parece que los 4 pasos anteriores quedan anulados con los siguientes pasos. Nota que pones "Abra IntelliJ" dos veces. ¿Les fata un subtítulo a los cuatro pasos anteriores y otro subtítulo a los siguientes pasos?

devarops replied:
Ya lo volví a hacer todo. Me parece que los primeros cuatro pasos sobran.

devarops commented on README.md:
```diff
@@ -0,0 +1,46 @@
+* Clone [este](https://github.com/IslasGECI/juego_10_nativos_dijeron.git) repositorio.
+* Abra intelliJ.
+* Abra el nuevo proyecto que se clonó.
+* Cuando el proyecto se haya abierto, agregue en la carpeta **src** el archivo "Juego.xlsx".
```
¿Dónde puedo encontrar `Juego.xlsx`?

devarops commented on README.md:
```diff
@@ -0,0 +1,46 @@
+* Abra intelliJ.
+* Cree un nuevo proyecto.
+* En la ventana de **new project** seleccione Java.
+* Seleccione una opción en **project sdk**, en caso de que no aparezca ninguna opción, da click en **new** y especifique la ruta en la que se encuentra el directorio del jdk de Java.
```
Creo que le falta un punto para separar en dos oraciones:
> Seleccione una opción en **project sdk**. En caso de que no aparezca ninguna opción, da click en **new** y especifique la ruta en la que se encuentra el directorio del jdk de Java.

devarops reviewed (changes requested):
En varios casos usas sólo minúsculas cuando en los mensajes aparece la primer letra en mayúscula. Por ejemplo _intelliJ_ debería ser _IntelliJ_, , _new project..._ debería ser _New Project..._,  _new_ debería ser _New..._

devarops commented:
Los guiones bajos (pisos) se ven raros en el título.

devarops reviewed (changes requested):
Por favor incluye una sunsección de instalación para Linux (ya que es el S.O. que usa el equipo). El comando para instalación en Ubunt es:
`sudo apt install default-jdk`

fiorellam committed:
Agrega instrucciones de instalación de Java en Linux

fiorellam committed:
Remueve instrucción de agregar archivo con la encuesta

Ya no es necesario agregar manualmente el archivo que contiene las preguntas y respuestas de la encuesta

devarops commented on README.md:
```diff
@@ -0,0 +1,47 @@
+* Clona [este](https://github.com/IslasGECI/juego_10_nativos_dijeron.git) repositorio.
+* Abre IntelliJ.
+* Abre el nuevo proyecto que se clonó.
+* Da click en el botón que tiene un recuadro rojo.
```
¿Qué comando representa ese ícono? En mi IDE no me aparece exactamente en el mismo lugar y hay dos íconos similares pero que ejecutan comandos distintos: uno es _Reimport all Maven Projects_ y el otro es _Syncronize_.
No sé a cual de los dos le debo picar

fiorellam replied:
Sería el de _Reimport all Maven Projects_

devarops replied:
Entonces hay que explicar esto en el `README`, no?

devarops commented on README.md:
```diff
@@ -0,0 +1,47 @@
+
+![image](https://user-images.githubusercontent.com/24512323/63110952-6cc2dc80-bf41-11e9-9d34-a472debf8c0b.png)
+
+* Da click en correr proyecto o en el botón con recuadro rojo que se puede encontrar en el archivo Main.java
```
No encuentro esto. Ni el botón ni el archivo `Main.java`

fiorellam replied:
Ok, me parece que se debe a que IntelliJ en ocasiones no reconoce la carpeta _src_ como la carpeta raíz, por lo que se debe hacer esto:
Se da click derecho a la carpeta _src_
![image](https://user-images.githubusercontent.com/24512323/63802400-67617c80-c8c7-11e9-8a48-b9194e2505db.png)

El archivo Main.java se encuentra aquí src/mx/org/islas/Main.java
![image](https://user-images.githubusercontent.com/24512323/63802510-a98abe00-c8c7-11e9-9030-2d085a645aa7.png)

devarops replied:
Entonces hay que explicar esto en el `README`, no?

fiorellam committed:
Agrega instrucciones para configurar IntelliJ
